lib/Runtime/Library/JsBuiltInEngineInterfaceExtensionObject.h

```diff
@@ -36,11 +43,16 @@ namespace Js
 
     private:
         Field(DynamicObject*) builtInNativeInterfaces;
-        Field(FunctionBody*) jsBuiltInByteCode;
 
-        Field(bool) wasInitialized;
+        #define BuiltInFunctionBody(class, type, obj) Field(FunctionBody*) jsBuiltIn##class##_##type##Bytecode = nullptr;
+        JsBuiltIns(BuiltInFunctionBody)
+        #undef BuiltInFunctionBody
```

Comment:
Create storage pointers for bytecode for each original Js file - using this mechanism adding more JS files with the correct names will extend this automatically.